### PR TITLE
LSS-Server-Status added

### PR DESCRIPTION
**Pull request by buLLzAT**
https://stats.uptimerobot.com/OEKDJSpmvK

### CODE REVIEW

File: src/LSSM-Menu.vue [hunk 1]
```diff
@@ -62,7 +62,12 @@
             </li>
             <li role="presentation">
                 <a class="lightbox-open" href="https://status.lss-manager.de/">
-                    Server Status
+                   LSSM-Server Status
+                </a>
+            </li>
+            <li role="presentation">
+                <a class="lightbox-open" href="https://stats.uptimerobot.com/3OVrETmBrM/786124522">
```
Comment (JRH-1997) [inline src/LSSM-Menu.vue:69]: This one is only for de_DE and the button is all games after you add that. 
You can use this link: https://stats.uptimerobot.com/OEKDJSpmvK for all games.

Comment (buLLzAT) [reply src/LSSM-Menu.vue:69]: oh yeah thx, I was just about to add the missing pages to my status page :)
Thx for the link

File: docs/de_DE/README.md [hunk 2]
```diff
@@ -11,7 +11,9 @@ sidebarDepth: 2
 
 <a :href="$themeConfig.variables.discord" target="_blank" style="float: right;"><img src="https://discord.com/api/guilds/254167535446917120/embed.png?style=banner1" alt="Unser Discord-Server: United Dispatch" data-prevent-zooming></a>
 
-[Server-Status](https://status.lss-manager.de)
+[LSSM-Server-Status](https://status.lss-manager.de)
+
+[LSS-Server-Status](https://stats.uptimerobot.com/3OVrETmBrM/786124522)
```
Comment (jxn-30) [inline docs/de_DE/README.md:16]: use https://stats.uptimerobot.com/OEKDJSpmvK here too

File: src/LSSM-Menu.vue [hunk 3]
```diff
@@ -62,7 +62,12 @@
...
+                   LSS-Server Status
```
Comment (jxn-30) [inline src/LSSM-Menu.vue:70]: Please remove that so we don't have to translate that ;)
Also when game is down you cannot access that link at all :)

**Review by jxn-30 — CHANGES_REQUESTED** [review]

**Comment by buLLzAT** [comment]
should be good now

**Review by jxn-30 — APPROVED** [review]

**Review by KokosnussDEV — APPROVED** [review]

File: docs/de_DE/README.md (hunk 2 above)
Comment (Piet2001) [reply docs/de_DE/README.md]: Where have i seen that link before?

Comment (jxn-30) [reply docs/de_DE/README.md]: Here: https://github.com/LSS-Manager/LSSM-V.4/pull/357#discussion_r541658491

Comment (JRH-1997) [reply docs/de_DE/README.md]: 😂 I don't know 😅

Comment (Piet2001) [reply docs/de_DE/README.md]: Ah, now, thanks @JRH-1997 for sharing my monitoring page 😛

**Review by jxn-30 — APPROVED** [review]

**Pull request merged**